mixer/pkg/attribute/protoBag.go

```diff
@@ -52,25 +50,33 @@ type ProtoBag struct {
 	referencedAttrsMutex sync.Mutex
 }
 
-// NewProtoBag creates a new proto-based attribute bag.
-func NewProtoBag(proto *mixerpb.CompressedAttributes, globalDict map[string]int32, globalWordList []string) *ProtoBag {
+var protoBags = sync.Pool{
+	New: func() interface{} {
+		return &ProtoBag{
+			referencedAttrs: make(map[attributeRef]mixerpb.ReferencedAttributes_Condition, 16),
```

Comment:
Do we recall why we chose `16`?  Should this be made a const, as it now is used in two places?

Reply:
@geeknoid  Thanks for the explanation.

Reply:
@douglas-reid  Have declared a const for `16`.